data/txHandler.go

```diff
@@ -310,6 +311,10 @@ func (handler *TxHandler) processIncomingTxn(rawmsg network.IncomingMessage) net
 	}
 	unverifiedTxGroup = unverifiedTxGroup[:ntx]
 
+	if ntx >= config.MaxTxGroupSize {
```

Comment:
For another PR, but these are easy checks for early rejection of invalid transactions.
Maybe we need a function to do these quick and dirty checks in here.

Reply:
I have it in dedup PR, I guess worth picking to here